src/services/players/player.cpp

```diff
@@ -127,7 +127,7 @@ namespace big
 
 	uint8_t player::id() const
 	{
-		return m_net_game_player == nullptr ? -1 : m_net_game_player->m_player_id;
+		return m_net_game_player == nullptr ? self::id : m_net_game_player->m_player_id;
 	}
```

Comment:
Wouldn't this change certain behaviour even if it's not our player object (for example when our target leaves online which may then end up targeting ourselves).

Reply:
Interesting point, but it should still fail safe instead of use undefined behavior. If that happens, and we revert it to how it was before (if is online, and m_net_game_player  is nullptr) then we would return -1, and the command would run like how it was before, where it would just spawn a car at 0,0,0.